plugin/PDFDocumentPage.qml

```diff
@@ -44,6 +45,11 @@ DocumentPage {
     PDF.Document {
         id: pdfDocument;
         source: base.path;
+        onDocumentLoaded: if (failure) {
+            pageStack.completeAnimation();
+            pageStack.pop();
+            notification.publish();
```

Comment:
This part I need to test on device to see if behavior can be improved, will take a bit more.